goshopify.go

```diff
@@ -343,11 +352,13 @@ func (c *Client) doGetHeaders(req *http.Request, v interface{}) (http.Header, er
 		return nil, respErr
 	}
 
+	c.doGetHeadersDebugResponse(resp)
```

Comment:
This will not log 500s and 429 responses, i think you should call this right after line 315.

Reply:
ah ya, that's fair.